modules/util_menu/menu.sql

```diff
@@ -40,7 +40,7 @@ CREATE TABLE `menu_entries` (
 
```

Comment:
I assume the changes in here are correct, but this utility needs to be put into the boneyard.